server/internal/gst/gst.go

```diff
@@ -99,9 +98,9 @@ func CreateAppPipeline(codecName string, pipelineDevice string, pipelineSrc stri
 		if pipelineSrc != "" {
 			pipelineStr = fmt.Sprintf(pipelineSrc+pipelineStr, pipelineDevice)
 		} else {
-			pipelineStr = fmt.Sprintf(videoSrc+"vp8enc cpu-used=8 threads=2 deadline=1 error-resilient=partitions keyframe-max-dist=10 auto-alt-ref=true"+pipelineStr, pipelineDevice)
+			pipelineStr = fmt.Sprintf(videoSrc+"vp8enc cpu-used=-5 threads=4 deadline=1 error-resilient=partitions keyframe-max-dist=30 auto-alt-ref=true"+pipelineStr, pipelineDevice)
```

Comment:
is there a reason behind changing `cpu-used=-5` to negative value?

Reply:
https://www.webmproject.org/docs/encoder-parameters/
"The meaning depends on the mode above. Negative values are for debug and force specific internal speed configurations."

I will set it back to 8? or 5? lower seems to be better quality but worse for the cpu.

Reply:
I don't know how it performs. Heavy CPU usage is kinda expected with transcoding, but if it increases quality, we can keep it in negative values then.